project/SparkBuild.scala

```diff
@@ -58,10 +58,10 @@ object BuildCommons {
 
   val allProjects@Seq(
     core, graphx, mllib, mllibLocal, repl, networkCommon, networkShuffle, launcher, unsafe, tags, sketch, kvstore,
-    commonUtils, sqlApi, _*
+    commonUtils, sqlApi, variant, _*
```

Comment:
Would also have to add at `dev/sparktestsupport/modules.py` and `.github/workflows/build_and_test.yml` so it runs the tests for individual modules.